packages/flame/src/Alert/Alert.tsx

```diff
@@ -100,31 +81,7 @@ export const Alert: React.FC<AlertProps & SpaceProps> = ({
           <Box fontSize="text-s">{children}</Box>
```

Comment:
Why a `Box` instead of `Text`? I would also add the corresponding `lineHeight` depending on size: https://design.lightspeedhq.com/components/Text/#text

Reply:
Cause it was a div in the initial implementation.

Same deal as the previous question/answer.